### ISSUE TITLE
update metric title as 'tokens per second' (TPS) rather than Words per Second (WPS)

### ISSUE THREAD

**Issue by lessw2020** [pytorch/torchtitan#263]
For some reason we are labelling tokens per second as WPS or words per second.
~~~
memory: 56.72GiB(59.68%)  wps: 5,739  mfu: 35.44%
~~~
This is not accurate. 
This issue is to track/discuss so we can update to TPS as the correct label.

**Comment by awgu** [comment]
I think we can close this since the `performance.md` includes the clarification about WPS and TPS now.

**Comment by eminorhan** [comment]
Also, it seems that the `wps` metric measures the per-device (local) tokens per second if I understand it correctly. It would be helpful to make this clear as this can be confusing (I first thought it was measuring the global tokens per second before looking at the code and running some examples myself).

**Comment by awgu** [comment]
@eminorhan what kind of clarification do you think would be helpful? e.g. in the readme or in the log line itself?

**Comment by eminorhan** [comment]
Definitely in the README but why not in the log line too? You do it for batch size (reporting both local and global) here, for instance: https://github.com/pytorch/torchtitan/blob/d2a4904f58accc683c17c66a360026cb3c8109af/train.py#L243

Something similar would be helpful for `wsp` too in my opinion. You could do something like:
```python
logger.info(f"global wps: {dp_degree} x {wps}")
```
to make it not too verbose.
